Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Shader/ShaderResourceGroup.h

```diff
@@ -157,7 +157,7 @@ namespace AZ
                 const RHI::BufferView* indirectResourceBuffer,
                 AZStd::span<const RHI::ImageView* const> imageViews,
                 uint32_t* outIndices,
-                bool viewReadOnly = true,
+                AZStd::span<bool> isViewReadOnly,
```

Comment:
Is this a breaking API change?  If so consider bumping the RPI `gem.json` to `1.0.0` and potentially adding an override for the deprecated method.